web_tool_script_3.R

```diff
@@ -216,44 +215,67 @@ if (file.exists(file.path(proc_input_path, portfolio_name, "overview_portfolio.r
 }
 
 
-indicies_equity_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_equity_portfolio.rda"))
-indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_bonds_portfolio.rda"))
-peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_equity_portfolio.rda"))
-peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_bonds_portfolio.rda"))
+indicies_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
+indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
+peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
+peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
 
 shock_year <- 2028 # this should come directly from the stress test
 
 create_interactive_report(
-  repo_path = repo_path,
-  template_dir = template_dir,
-  company_charts_dir = company_charts_dir,
+  repo_path,
```

Comment:
this should have named arguments, see here https://github.com/2DegreesInvesting/PACTA_analysis/blob/48eeea1d9e9e70e54f3b22dfc5075976892cf363/web_tool_script_3.R#L227

Reply:
updated all here